### ISSUE TITLE
Implement an AMQP dev service

### ISSUE THREAD

**Issue by vkasala** [amqphub/quarkus-qpid-jms#15]
From Quarkus 2.0, the Dev Service support is increasing for each extensions.

Implement an AMQP dev service similar to https://github.com/quarkusio/quarkus/pull/17844/files

**Comment by cescoffier** [comment]
Quarkus Core as an AMQP 1.0 dev service (using Artemis). We could extract the classes to reuse the same dev service.

**Comment by zhfeng** [comment]
@cescoffier is there any quarkus issue trancking to extract the classes for reusing? I think we have a similar one with https://github.com/quarkiverse/quarkus-artemis/issues/54

**Comment by middagj** [comment]
Note that Artemis has a multi port acceptor on port 61616 which can be used for Core (Artemis JMS), OpenWire, STOMP, AMQP 1.0, MQTT and HornetQ.

**Comment by cescoffier** [comment]
I don't believe there is a Quarkus issue per-se. However having multiple similar dev service could be a disaster if the extensions are used together.
